theano/gpuarray/dnn.py

```diff
@@ -1510,20 +1510,23 @@ def local_dnn_convgi_inplace(node, inputs):
 
 @register_opt('cudnn')
 @alpha_merge(GpuDnnConv, alpha_in=4, beta_in=5)
-def local_dnn_conv_alpha_merge(node, *inputs):
-    return [gpu_dnn_conv(algo=node.op.algo)(*inputs)]
+@register_opt2([GpuDnnConv], 'fast_compile', 'cudnn')
+def local_dnn_conv_alpha_merge(op, context_name, inputs, outputs):
+    return [gpu_dnn_conv(algo=op.algo)(*inputs)]
```

Comment:
I don't understand why you changed the alpha_merge and output_merge opt. Those aren't op_lifter. Currently, with the way GraphToGPU is made, those optimization  won't get applied in GraphToGPU. You can let those be done in the old opt.